### DISCUSSION TITLE
Remove background - make transparent

### DISCUSSION THREAD

**Discussion by karantzak** [ImageMagick/ImageMagick#3496] [Help]
I want to remove the backgroung of my images and make them transparent.
The current background is not just one color (i.e white) but it has a lot of variations of grey and white (see the attached image) 
How can I make it transparent with your tool?

![photo_4_11_smal_0x90](https://user-images.githubusercontent.com/81800494/113395221-2f26f780-93a2-11eb-8b47-610c54fc93f7.jpg)

**Comment by snibgo** [comment]
If you want a fully automatic solution, I don't think ImageMagick can help you here. In places, the sides of the bottle have the same colour as the background. So IM cannot detect which pixels are background and which are foreground.

**Comment by fmw42** [comment]
Try using http://remove.bg

**Comment by karantzak** [comment]
Thanks for your replies
